app/build.gradle.kts

```diff
@@ -63,16 +63,13 @@ android {
 
     flavorDimensions += "product"
     productFlavors {
-        create("dark") {
+        create("app") {
```

Comment:
have used name app, let me know if we can use other variant name like `default` or anyother

Reply:
It's fine.